event_participant.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy import Column, Integer, VARCHAR, DateTime
+from db_handler import Base
+from sqlalchemy import ForeignKey
+from sqlalchemy.orm import relationship, backref
+
+
+class EventParticipant(Base):
```

Comment:
There is an easier way to define association tables.

Reply:
Removed redundant code